apollo-router/src/plugins/response_cache/postgres.rs

```diff
@@ -115,6 +122,10 @@ pub(crate) enum PostgresCacheStorageError {
     BadConfiguration(String),
     #[error("postgres error: {0}")]
     PgError(#[from] sqlx::Error),
+    #[error("cleanup_interval configuration is out of range: {0}")]
+    OutOfRangeError(#[from] chrono::OutOfRangeError),
+    #[error("cleanup_interval configuration is incorrect: {0}")]
+    WrongCleanupInterval(String),
```

Comment:
Nit: this is very minor semantics, but I think incorrect/wrong imply that there's one 'correct' interval rather than a huge range of options

```suggestion
    #[error("cleanup_interval configuration is invalid: {0}")]
    InvalidCleanupInterval(String),
```